src/fragmentarium/ui/display/Transliteration.tsx

```diff
@@ -143,16 +151,60 @@ function DisplayLine({
   )
 }
 
+function DollarLineAndAtLine({
+  line,
+  container = 'div'
+}: {
+  line: LineContainer
+  container?: string
+}): JSX.Element {
+  if (line.type === 'RulingDollarLine') {
+    const rulingDollarLine = line as RulingDollarLine
+    return React.createElement(
+      container,
+      { className: classNames([`Transliteration__ControlLine`]) },
+      [
+        <hr
+          key={101}
+          className={`Transliteration__${line.type} ${rulingDollarLine.number}`}
+        />
+      ]
+    )
+  } else {
+    return React.createElement(
+      container,
+      { className: classNames([`Transliteration__ControlLine`]) },
+      [
+        <div key={101} className="Transliteration__DollarLineAndAtLine">
+          ({line.content[0].value.trimStart()})
```

Comment:
Do not use `content` it will be removed once all fragments are updated to the newest state. Dollar- and at-lines have the proper model you implemented earlier. If it seems to bee too complicated to generate the content in the fronted (e.g. state lines) consider adding a property for display value in the API.